features/buyer/award_flow_requirements.feature

```diff
@@ -74,6 +91,23 @@ Scenario: Award flow - Mark requirements as unsuccessful
   And I see a temporary-message banner message containing 'The buyer didn't award this contract because no suppliers met their '
   And I see a temporary-message banner message containing 'requirements. They may publish an updated version later.'
 
+@skip-staging
+Scenario: Award flow - Mark requirements as unsuccessful
+
+  Given I choose 'No' radio button
+  And I click the 'Save and continue' button
+  Then I am on the 'Why didn’t you award a contract for %s?' page with brief 'title'
+
+  When I choose 'There were no suitable suppliers' radio button
+  And I click the 'Update requirements' button
+  Then I am on the '%s' page with brief 'title'
+  And I see a success banner message containing 'updated'
+
+  When I go to that brief page
+  Then I see a temporary-message banner message containing 'No suitable suppliers applied'
+  And I see a temporary-message banner message containing 'The buyer didn't award this contract because no suppliers met their '
```

Comment:
Just to confirm, this message appears on the Buyer FE, right? So no apostrophe change needed yet for `The buyer didn't`?

Reply:
Yep!